### ISSUE TITLE
fix: useFetch's does not implement static target (payload is not saved) 

### ISSUE THREAD

**Issue by gnuletik** [nuxt-community/composition-api#127]
**🐛 The bug**
When using `useFetch` hook and generating a full-static build (target: static) and `nuxt export`, the useFetch payload is not saved. 

**🛠️ To reproduce**

pages/test.vue
```typescript
export default defineComponent({
  setup () {
    const name = ref('')

    useFetch(async () => {
      name.value = await axios.get('https://myapi.com/name')
    })
  }
})
```

nuxt.config.js
```javascript
export default {
  target: 'static'
}
```
`yarn run nuxt build && yarn run nuxt export  && http-server dist && xdg-open localhost:8080`

When I load index and navigate to the page `test`, I can see the network request in the Network tab of the browser.

**🌈 Expected behaviour**

The HTTP request should not be executed on client-side.

**ℹ️ Additional context**

I know there is `useStatic` but it does not allow to set a server target (if you want to make a second build which get data from client-side), does not support `enablePreview`, and other features implemented by [nuxt static mode](https://nuxtjs.org/blog/going-full-static).
Also, `useStatic` requires code changes (which is specific to this package) instead of the `useFetch` implementation which has the same behavior as Nuxt's Options API.

Btw, thanks for this module :) It's awesome to combine the Composition API with Nuxt ! :tada:

**Comment by gnuletik** [comment]
My bad, this issue comes from Nuxt :)
https://github.com/nuxt/nuxt.js/issues/7285

**Comment by danielroe** [comment]
@gnuletik Thanks for the positive feedback 😊

I've just released 0.10.0 with support for full static in `useFetch`. Let me know if it works for you!
